### PR TITLE
Review and add tips from notion doc

### PR DESCRIPTION
**Pull request by J2-D2-3PO**
Pulls relevant info from best practices docs linked in https://wandb.atlassian.net/browse/DOCS-838, and adds it to various areas in the docs. Left over from the Jan docs team 🐛 🔨 

Confirm that tests pass and this change is ready for review:
- [x] Local `hugo serve`
- [x] Local `vale ./content`

### CODE REVIEW

File: content/guides/models/track/runs/_index.md [hunk 1]
```diff
@@ -403,6 +403,28 @@ Delete one or more runs from a project with the W&B App.
 For projects that contain a large number of runs, you can use either the search bar to filter runs you want to delete using Regex or the filter button to filter runs based on their status, tags, or other properties. 
 {{% /alert %}}
 
+## Organize runs 
+
+This section provides instructions on how to organize runs using groups and job types. By assigning runs to groups (e.g., experiment names) and specifying job types (e.g., preprocessing, training, evaluation, debugging), you can streamline your workflow and improve model comparison.
```
[the remark this answers is not in the document]
Comment (J2-D2-3PO) [reply content/guides/models/track/runs/_index.md]: Addressed in https://github.com/wandb/docs/pull/1053/commits/764f435f888a06354837d2869524e0c3d7a5a053

File: content/guides/core/reports/edit-a-report.md [hunk 2]
```diff
@@ -394,4 +394,11 @@ Select a panel grid and press `delete` on your keyboard to delete a panel grid.
...
 
-{{< img src="/images/reports/collapse_headers.gif" alt="" >}}
\ No newline at end of file
+{{< img src="/images/reports/collapse_headers.gif" alt="" >}}
+
+## Visualize relationships across multiple dimensions
+
+To effectively visualize relationships across multiple dimensions, use a color gradient to represent one of the variables. This enhances clarity and makes patterns easier to interpret.
+
+1. Choose a variable to represent with a color gradient (e.g., penalty scores, learning rates, etc.). This allows for a clearer understanding of how penalty (color) interacts with reward/side effects (y-axis) over training time (x-axis).
```
[the remark this answers is not in the document]
Comment (J2-D2-3PO) [reply content/guides/core/reports/edit-a-report.md:403]: Addressed in https://github.com/wandb/docs/pull/1053/commits/764f435f888a06354837d2869524e0c3d7a5a053

File: content/guides/core/artifacts/_index.md [hunk 3]
```diff
@@ -65,14 +65,18 @@ Indicate the artifact you want to mark as input to your run with the [`use_artif
 Following the preceding code snippet, this next code block shows how to use the `training_dataset` artifact: 
 
 ```python
-artifact = run.use_artifact("training_dataset:latest") #returns a run object using the "my_data" artifact
+artifact = run.use_artifact(
+    "training_dataset:latest"
+)  # returns a run object using the "my_data" artifact
```
Comment (mdlinville) [inline content/guides/core/artifacts/_index.md:70]: Is this comment out of sync? I don't see `my_data` anywhere, but maybe I'm just confused. Same feedback for line 79.

File: content/guides/core/artifacts/_index.md [hunk 4]
```diff
@@ -65,14 +65,18 @@ Indicate the artifact you want to mark as input to your run with the [`use_artif
...
 ```
 This returns an artifact object.
 
 Next, use the returned object to download all contents of the artifact:
 
 ```python
-datadir = artifact.download() #downloads the full "my_data" artifact to the default directory.
+datadir = (
+    artifact.download()
+)  # downloads the full "my_data" artifact to the default directory.
```
Comment (mdlinville) [inline content/guides/core/artifacts/_index.md]: Not sure "my_data" should be quoted -- we are in a code block.

File: content/guides/core/reports/_index.md [hunk 5]
```diff
@@ -50,4 +50,8 @@ Depending on your use case, explore the following resources to get started with
 * Check out our [video demonstration](https://www.youtube.com/watch?v=2xeJIv_K_eI) to get an overview of W&B Reports.
 * Explore the [Reports gallery]({{< relref "./reports-gallery.md" >}}) for examples of live reports.
 * Try the [Programmatic Workspaces]({{< relref "/tutorials/workspaces.md" >}}) tutorial to learn how to create and customize your workspace.
-* Read curated Reports in [W&B Fully Connected](http://wandb.me/fc).
\ No newline at end of file
+* Read curated Reports in [W&B Fully Connected](http://wandb.me/fc).
+
+## Best practices and tips
```
Comment (mdlinville) [inline content/guides/core/reports/_index.md]:
```suggestion
## Recommended practices and tips 
```
"Best" is in the eye of the beholder, so it can be safer to avoid using the term "best practice" too prominently. NOt sure if we have any rules here.

File: content/guides/core/reports/edit-a-report.md [hunk 6]
```diff
@@ -394,4 +394,11 @@ Select a panel grid and press `delete` on your keyboard to delete a panel grid.
 
 Collapse headers in a Report to hide content within a text block. When the report is loaded, only headers that are expanded will show content. Collapsing headers in reports can help organize your content and prevent excessive data loading. The proceeding gif demonstrates the process.
 
-{{< img src="/images/reports/collapse_headers.gif" alt="" >}}
\ No newline at end of file
+{{< img src="/images/reports/collapse_headers.gif" alt="" >}}
```
Comment (mdlinville) [inline content/guides/core/reports/edit-a-report.md]: Please give alt text, it's a shame to explicitly leave it blank.

File: content/guides/core/reports/edit-a-report.md [hunk 7]
```diff
@@ -394,4 +394,11 @@ Select a panel grid and press `delete` on your keyboard to delete a panel grid.
...
+
+## Visualize relationships across multiple dimensions
```
Comment (mdlinville) [inline content/guides/core/reports/edit-a-report.md:399]: Relationships of what?  Maybe "dimension" means a specific thing here and it is clearer to customers than to me.

File: content/guides/models/track/_index.md [hunk 8]
```diff
@@ -59,4 +59,8 @@ Depending on your use case, explore the following resources to get started with
   * Configure experiments
   * Log data from experiments
   * View results from experiments
-* Explore the [W&B Python Library]({{< relref "/ref/python/" >}}) within the [W&B API Reference Guide]({{< relref "/ref/" >}}).
\ No newline at end of file
+* Explore the [W&B Python Library]({{< relref "/ref/python/" >}}) within the [W&B API Reference Guide]({{< relref "/ref/" >}}).
```
Comment (mdlinville) [inline content/guides/models/track/_index.md:62]: Do we call it the Python library or the API?

File: content/guides/models/track/_index.md [hunk 9]
```diff
@@ -59,4 +59,8 @@ Depending on your use case, explore the following resources to get started with
...
+
+## Best practices and tips
```
Comment (mdlinville) [inline content/guides/models/track/_index.md:64]: Similar feedback about "best practices". Also a nit, I don't think Experiments or Logging need to be capitalized in line 66. It seems especially weird to capitalize Experiments, not capitalize logging, then capitalize both in the link text!

File: content/guides/models/track/launch.md [hunk 10]
```diff
@@ -127,6 +127,7 @@ The following are some suggested guidelines to consider when you create experime
 2. **Project**: A project is a set of experiments you can compare together. Each project gets a dedicated dashboard page, and you can easily turn on and off different groups of runs to compare different model versions.
 3. **Notes**: Set a quick commit message directly from your script. Edit and access notes in the Overview section of a run in the W&B App.
 4. **Tags**: Identify baseline runs and favorite runs. You can filter runs using tags. You can edit tags at a later time on the Overview section of your project's dashboard on the W&B App.
+5. **Create multiple run sets for easy comparison**: When comparing experiments, create multiple run sets to make metrics easy to compare. You can toggle run sets on or off on the same chart or group of charts.
```
Comment (mdlinville) [inline content/guides/models/track/launch.md]:
```suggestion
5. **Create multiple run sets to compare experiments**: When comparing experiments, create multiple run sets to make metrics easy to compare. You can toggle run sets on or off on the same chart or group of charts.
```
Who knows if it's easy or not, and "for easy comparison" is a bit passive.

File: content/guides/models/track/log/_index.md [hunk 11]
```diff
@@ -81,4 +81,8 @@ wandb.log({"loss": 0.314, "epoch": 5,
 1. **Compare the best accuracy**: To compare the best value of a metric across runs, set the summary value for that metric. By default, summary is set to the last value you logged for each key. This is useful in the table in the UI, where you can sort and filter runs based on their summary metrics, to help compare runs in a table or bar chart based on their _best_ accuracy, instead of final accuracy. For example: `wandb.run.summary["best_accuracy"] = best_accuracy`
 2. **Multiple metrics on one chart**: Log multiple metrics in the same call to `wandb.log`, like this: `wandb.log({"acc'": 0.9, "loss": 0.1})` and they will both be available to plot against in the UI
```
Comment (mdlinville) [inline content/guides/models/track/log/_index.md]: This and the next entry have incomplete sentence in bold, but the first and 4th have complete  cta-type sentences. Maybe "View multiple metrics in a single chart" and "Customize the X axis" or something like that?

File: content/guides/models/track/runs/_index.md [hunk 12]
```diff
@@ -468,6 +468,28 @@ Delete one or more runs from a project with the W&B App.
 For projects that contain a large number of runs, you can use either the search bar to filter runs you want to delete using Regex or the filter button to filter runs based on their status, tags, or other properties. 
 {{% /alert %}}
 
+## Organize runs 
+
+This section provides instructions on how to organize runs using groups and job types. By assigning runs to groups (for example, experiment names) and specifying job types (for example, preprocessing, training, evaluation, debugging), you can streamline your workflow and improve model comparison.
+
+### Assigning runs to groups and job types
```
Comment (mdlinville) [inline content/guides/models/track/runs/_index.md]:
```suggestion
### Assign a group or job type to a run
```

File: content/guides/models/track/runs/_index.md [hunk 13]
```diff
@@ -468,6 +468,28 @@ Delete one or more runs from a project with the W&B App.
...
+
+Each run in W&B can be categorized by **group** and a **job type**:
+
+- **Group**: Represents a broader experiment category, making it easier to organize and filter runs.
+- **Job type**: Describes the function of the run, such as preprocessing, training, or evaluation.
+
```
Comment (mdlinville) [inline content/guides/models/track/runs/_index.md]:
```suggestion
- **Group**: a broad category for the experiment, used to organize and filter runs.
- **Job type**: the function of the run, such as `preprocessing`, `training`, or `evaluation`.

```

File: content/guides/models/track/runs/_index.md [hunk 14]
```diff
@@ -468,6 +468,28 @@ Delete one or more runs from a project with the W&B App.
...
+In the following [example workspace](https://wandb.ai/stacey/model_iterz?workspace=user-stacey), a baseline model is trained using increasing amounts of data from the Fashion-MNIST dataset. The color coding in the workspace represents the amount of data used:
```
Comment (mdlinville) [inline content/guides/models/track/runs/_index.md]:
```suggestion
The proceeding [example workspace](https://wandb.ai/stacey/model_iterz?workspace=user-stacey), trains a baseline model using increasing amounts of data from the Fashion-MNIST dataset. The workspace uses colorts to represent the amount of data used:
```
Maybe link to the dataset

File: content/guides/models/track/runs/_index.md [hunk 15]
```diff
@@ -468,6 +468,28 @@ Delete one or more runs from a project with the W&B App.
...
+
+- **Yellow to dark green**: Increasing amounts of data for the baseline model.
```
Comment (mdlinville) [inline content/guides/models/track/runs/_index.md]:
```suggestion
- **Yellow to dark green** indicate increasing amounts of data for the baseline model.
```

File: content/guides/models/track/runs/_index.md [hunk 16]
```diff
@@ -468,6 +468,28 @@ Delete one or more runs from a project with the W&B App.
...
+- **Light blue to violet to magenta**: Increasing amounts of data for a more complex "double" model with additional parameters.
```
Comment (mdlinville) [inline content/guides/models/track/runs/_index.md]:
```suggestion
- **Light blue to violet to magenta** indicate amounts of data for a more complex "double" model with additional parameters.
```

File: content/guides/models/track/runs/_index.md [hunk 17]
```diff
@@ -468,6 +468,28 @@ Delete one or more runs from a project with the W&B App.
...
+
+Using W&B's filtering options and search bar, you can easily compare runs based on specific conditions, such as:
```
Comment (mdlinville) [inline content/guides/models/track/runs/_index.md]:
```suggestion
Use W&B's filtering options and search bar to compare runs based on specific conditions, such as:
```

File: content/guides/models/track/runs/_index.md [hunk 18]
```diff
@@ -468,6 +468,28 @@ Delete one or more runs from a project with the W&B App.
...
+- Training on the same dataset.
+- Evaluating on the same test set.
+
+Applying filters dynamically updates the **Table** view, allowing you to quickly identify performance differences between models. For example, you can determine which classes are significantly more challenging for one model compared to another.
```
Comment (mdlinville) [inline content/guides/models/track/runs/_index.md]:
```suggestion
When you apply filters, the **Table** view is updated automatically. This allows you to identify performance differences between models, such as determining which classes are significantly more challenging for one model compared to another.
```

**Review by mdlinville — APPROVED** [review]
Some copyedit suggestions to use more active text avoid stating opinions about what is easy or difficult. If a suggestion doesn't help, please ignore it. I'll go ahead and approve so that you won't be blocked either way.

File: content/guides/core/artifacts/_index.md (hunk 4 above)
Comment (J2-D2-3PO) [inline content/guides/core/artifacts/_index.md]:
```suggestion
)  # downloads the full `my_data` artifact to the default directory.
```

File: content/guides/core/reports/edit-a-report.md (hunk 6 above)
Comment (J2-D2-3PO) [inline content/guides/core/reports/edit-a-report.md]:
```suggestion
{{< img src="/images/reports/collapse_headers.gif" alt="Collapsing headers in a report." >}}
```

File: content/guides/models/track/_index.md (hunk 8 above)
Comment (J2-D2-3PO) [reply content/guides/models/track/_index.md:62]: Looks like we call it "Library" in the docs, will defer to that https://docs.wandb.ai/ref/python/

File: content/guides/models/track/_index.md [hunk 19]
```diff
@@ -59,4 +59,8 @@ Depending on your use case, explore the following resources to get started with
   * Configure experiments
   * Log data from experiments
   * View results from experiments
-* Explore the [W&B Python Library]({{< relref "/ref/python/" >}}) within the [W&B API Reference Guide]({{< relref "/ref/" >}}).
\ No newline at end of file
+* Explore the [W&B Python Library]({{< relref "/ref/python/" >}}) within the [W&B API Reference Guide]({{< relref "/ref/" >}}).
+
+## Best practices and tips 
+
+For best practices and tips for Experiments and logging, see [Best Practices: Experiments and Logging](https://wandb.ai/wandb/pytorch-lightning-e2e/reports/W-B-Best-Practices-Guide--VmlldzozNTU1ODY1#w&b-experiments-and-logging).
```
Comment (J2-D2-3PO) [inline content/guides/models/track/_index.md]:
```suggestion
For best practices and tips for experiments and logging, see [Best Practices: Experiments and Logging](https://wandb.ai/wandb/pytorch-lightning-e2e/reports/W-B-Best-Practices-Guide--VmlldzozNTU1ODY1#w&b-experiments-and-logging).
```

File: content/guides/models/track/_index.md (hunk 9 above)
Comment (J2-D2-3PO) [reply content/guides/models/track/_index.md:64]: Agreed on first instance of Experiments, I lowercased it. However, the other instance just reflects the title from the report, so deferring to that

File: content/guides/models/track/log/_index.md (hunk 11 above)
Comment (J2-D2-3PO) [inline content/guides/models/track/log/_index.md]:
```suggestion
2. **View multiple metrics on one chart**: Log multiple metrics in the same call to `wandb.log`, like this: `wandb.log({"acc'": 0.9, "loss": 0.1})` and they will both be available to plot against in the UI
```

File: content/guides/models/track/log/_index.md [hunk 20]
```diff
@@ -81,4 +81,8 @@ wandb.log({"loss": 0.314, "epoch": 5,
...
 3. **Custom x-axis**: Add a custom x-axis to the same log call to visualize your metrics against a different axis in the W&B dashboard. For example: `wandb.log({'acc': 0.9, 'epoch': 3, 'batch': 117})`. To set the default x-axis for a given metric use [Run.define_metric()]({{< relref "/ref/python/run.md#define_metric" >}})
```
Comment (J2-D2-3PO) [inline content/guides/models/track/log/_index.md]:
```suggestion
3. **Customize the x-axis**: Add a custom x-axis to the same log call to visualize your metrics against a different axis in the W&B dashboard. For example: `wandb.log({'acc': 0.9, 'epoch': 3, 'batch': 117})`. To set the default x-axis for a given metric use [Run.define_metric()]({{< relref "/ref/python/run.md#define_metric" >}})
```

File: content/guides/core/artifacts/_index.md (hunk 3 above)
Comment (J2-D2-3PO) [reply content/guides/core/artifacts/_index.md:70]: It comes from the dataset

File: content/guides/core/artifacts/_index.md (hunk 4 above)
Comment (J2-D2-3PO) [reply content/guides/core/artifacts/_index.md]: I changed it to backticks to imply that it's a code object

File: content/guides/core/reports/edit-a-report.md (hunk 6 above)
Comment (J2-D2-3PO) [reply content/guides/core/reports/edit-a-report.md]: Added

File: content/guides/models/track/log/_index.md (hunk 11 above)
Comment (J2-D2-3PO) [reply content/guides/models/track/log/_index.md]: Agreed, updated

**Comment by J2-D2-3PO** [comment]
> Some copyedit suggestions to use more active text avoid stating opinions about what is easy or difficult. If a suggestion doesn't help, please ignore it. I'll go ahead and approve so that you won't be blocked either way.

Thanks I think I addressed most everything
